taxi/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
 
-# Create your views here.
+from django.http import HttpResponse, HttpRequest
+
+from taxi.models import Driver, Manufacturer, Car
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
```

Comment:
There is no need to create these variables, you could pas count operation directly as values in the dict